script.js

```diff
@@ -0,0 +1,150 @@
+const toDoForm = document.querySelector(".toDoForm");
+const toDoInput = toDoForm.querySelector("input");
+const toDoList = document.querySelector(".toDoList");
+const finishedList = document.querySelector(".finishedList");
+const lenTodo = document.getElementById("todo-count");
+const lenFin = document.getElementById("fin-count");
```

Comment:
변수의 네이밍이 조금 모호한 느낌이 들어요. 딱 봤을 때 무엇을 뜻하는지 모르겠는 느낌? 한번 이렇게 고쳐보면 어떨까요? 사실 제 생각이라 좀 더 좋은 이름이 있으면 제안해주세요!
```suggestion
const toDoLength = document.getElementById("todo-count");
const finLength = document.getElementById("fin-count");
```

Reply:
훨씬 좋은 것 같아요! 수정하겠습니다!